lib/reactotron-react-native/src/plugins/networking.ts

```diff
@@ -1,6 +1,65 @@
-import XHRInterceptor from "react-native/Libraries/Network/XHRInterceptor"
 import type { ReactotronCore, Plugin } from "reactotron-core-client"
 
+function getXhrInterceptorPath() {
+  const defaultPath = "react-native/Libraries/Network/XHRInterceptor"
+  const newPath = "react-native/src/private/inspector/XHRInterceptor"
+
+  try {
+    const reactNativeVersionString = require("react-native/package.json").version
+    const match = reactNativeVersionString.match(/^(\d+)\.(\d+)\./)
```

Comment:
I've seen this technique used in expo-config plugins, should work here https://github.com/expo/config-plugins/blob/ec4f114b825b85e5cd42b42205a9c0a99097098f/packages/detox/src/withDetox.ts#L51-L60

Reply:
Ended up going with a different approach